platform.yaml

```diff
@@ -14,7 +14,7 @@ spec:
     spec:
       containers:
       - name: price-model
-        image: pocketsizefund-{ENVIRONMENT}-price-model:latest
```

Comment:
I think this makes sense - the only reason I included an environment differentiator was to make sure a broken image wasn't pushed to the repository to then break prod or anything. There are better ways than that for handling it though.